gemfileparser/__init__.py

```diff
@@ -58,12 +60,14 @@ def __init__(self, filepath, appname=''):
         self.dependencies = {
             'development': [],
             'runtime': [],
+            'dependency': [],
             'test': [],
             'production': [],
             'metrics': []
         }
         self.contents = self.gemfile.readlines()
-        if filepath.endswith('gemspec'):
+        path = ('gemspec', 'podspec')
+        if filepath.endswith(path):
```

Comment:
`path` is IMHO not a happy variable name as it could shadow stdlib imports .. and you further never reuse that afterwards... 
So I suggest either:
1. to use a better name such as `spec_extensions` (note the plural) as in :
` path = ('gemspec', 'podspec', )` (note also the trailing comma that is what makes this a tuple and is more explicit.)
2. or just use directly this `if filepath.endswith(('gemspec', 'podspec',)):`

Finally since we really care bout extensions, what about adding a leading period as in:
`if filepath.endswith(('.gemspec', '.podspec',)):`

Reply:
Okay :+1:
I will make changes in few minutes :)